### PR TITLE
Allow to use switch NI ports as VLAN access ports

### PR DESCRIPTION
**Pull request by milan-zededa**
The main use-case of switch NI with multiple ports is not actually to
bridge multiple L2 switches for redundancy and run STP (*) to avoid loops,
but to allow connecting external end devices (e.g. sensors) directly
into the same L2 segment as the applications running on the edge node.
With VLAN-enabled Switch NI, the user should be able to use a physical
network port attached to Switch NI as a VLAN access port (by default
all switch NI ports are configured as trunks).

(*) Regardless of the use-case, we will always run STP on a multi-port
switch NI to avoid accidental bridge loops.

### CODE REVIEW

File: proto/config/netinst.proto [hunk 1]
```diff
@@ -310,4 +310,21 @@ message NetworkInstanceConfig {
...
+  repeated VlanAccessPort vlan_access_ports = 51;
 }
+
+// VLAN Access Port config applied to physical port(s) attached to a Switch Network Instance.
+message VlanAccessPort {
+  // VLAN ID from the range <1,4094>
+  uint32 vlan_id = 1;
+  // Physical network adapter used as the access port for the given VLAN.
+  // Either a single NI port referenced by its name (SystemAdapter.Name, aka logical label)
+  // or an adapter shared-label matching zero or more NI ports.
```
Comment (eriknordmark) [inline proto/config/netinst.proto:328]: Does it make sense even when it matches zero ports? Is this so it will wait for matching ports to appear later?

Comment (milan-zededa) [reply proto/config/netinst.proto:328]: This is just a generic description for shared labels.
Config that never matches any port is indeed pointless. However, there can be a case where temporarily no port is matched but later a new port appears (that was previously e.g. assigned to an app) or user adds shared label to an existing port. Zedrouter will then add the port into the switch NI and apply the VLAN config.

Comment (eriknordmark) [reply proto/config/netinst.proto:328]: ok

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**